File: Unidad 3/Meta 3.1/Java/Biosenosis.java
public class Biosenosis {
	private String nombre;
	private Ecosistema ecosistema;

	public Biosenosis(String nombre) {
		this.nombre = nombre;
	}

	public void setEcosistema(Ecosistema ecosistema) {
		this.ecosistema = ecosistema;
	}

	public Ecosistema getEcosistema() {
		return ecosistema;
	}

	public String getNombre() {
		return nombre;
	}

	public void setNombre(String nombre) {
		this.nombre = nombre;
	}

	@Override
    public String toString() {
        return "Biosenosis{" + "nombre=" + nombre + '}';
    }

}

File: Unidad 2/Meta 2.1/C#/Ecosistema/Biosenosis.cs
using System;

public class Biosenosis
{
    private string nombre;

    public Biosenosis(string nombre)
    {
        this.nombre = nombre;
    }

    public string Nombre
    {
        get { return this.nombre; }
        set { this.nombre = value; }
    }

    override public string ToString()
    {
        return "Biosenosis{" + "nombre=" + this.nombre + '}';
    }
}

